test/system-tests/kitchen.ts

```diff
@@ -3,13 +3,15 @@ import * as fs from 'fs';
 import * as r from 'request';
 const upload = require('../../src');
 
+const bucketName = process.env.BUCKET_NAME || 'stephen-has-a-new-bucket';
```

Comment:
I can just put `stephen-has-a-new-bucket` as an env var locally, don't worry about keeping that here :D